.github/workflows/build-and-test.yml

```diff
@@ -19,9 +11,7 @@ jobs:
         uses: actions/setup-go@v3.3.0
         with:
           go-version-file: go.mod
-      - name: Install goimports
-        run: go install golang.org/x/tools/cmd/goimports@v0.3.0
       - name: Build
         run: make build
```

Comment:
Maybe this is not needed

Reply:
you mean the "make build"?
If so indeed

Reply:
yes, the make build